src/commands/upload_file.ts

```diff
@@ -253,16 +253,23 @@ new CLICommand({
 					driveKey,
 					drivePassword
 				} of filesToUpload) {
+					let driveKeyForStats = driveKey;
+					if (!driveKeyForStats) {
+						const driveSignatureInfo = await arDrive.getDriveSignatureInfo(
```

Comment:
let's update the SDK function to take an object parameter with named keys so that users are less likely to pass a wallet address for a driveID and vice versa.

Reply:
updated ardrive-core-js to add object parameter and updated code here in https://github.com/ardriveapp/ardrive-cli/pull/361/commits/5d83853d6f96ffa52f58d925c6ddab438b9d5289